.pylintrc

```diff
@@ -70,6 +70,9 @@ disable=fixme,              # disabled as TODOs would show up as warnings
     no-else-return,     # relax "elif" after a clause with a return
     docstring-first-line-empty, # relax docstring style
     import-outside-toplevel,
+    cyclic-import,  # This checker raises on all module pairs that import each other,
```

Comment:
Should we also turn off parallelism when running pylint to catch more errors potentially now that this check has been disabled?

Reply:
I am not sure what else there is that can get caught with parallelism disabled. Our tests are slower so there is not much downside to removing parallelism for the CI runs (I see around 10 mins for test and under 3 for pylint with parallelism). Maybe it should be left on for a run someone would call locally, or at least easy to opt into. Or maybe people don't generally run it on all files locally any way (versus just on the file being edited). What do you think?

Reply:
I added a commit to remove the parallelism but make it possible to enable it optionally with tox by moving the `posargs` location. I don't think it's common to want extra arguments to black?

Reply:
Thanks, that looks good! I haven't heard of people passing arguments to black. And I don't know specifically what removing parallelism might catch, but it seems fine since linting isn't that slow.

Reply:
I actually ran into a reason to pass an argument to black. tox runs with `--check` without `--diff`. You might want to run with `--diff` to see what the changes would be. It's simple enough just to run `-eblack --diff` in that case though.

Reply:
By the way, the time only changed from 2:46 to 2:54 for the two runs here with and without parallelism. There could be more difference locally on a system with more cores. The GitHub runners seem to have only two.